src/core/utils.js

```diff
@@ -722,6 +722,19 @@ export function sanitizeUrl(url) {
   return braintreeSanitizeUrl(url)
 }
 
+export function isAbsoluteUrl(url) {
+  return url.match(/^(?:[a-z]+:)?\/\//i) // Matches http://, HTTP://, https://, ftp://, //example.com,
+}
+
+export function buildUrl(url="", selectedServer="") {
```

Comment:
**Change request**: new util functions need accompanying tests - ideally that would include cases for expected outputs, graceful handling of malformed inputs, and basic type safety (esp. `undefined`/`null`).

Reply:
it's not that I don't trust your code @geraldglynn - it's that I don't trust myself to remember all the edge cases relevant to this function when a first-time contributor opens a PR that touches it in 10 months 😄